nomt/src/rw_pass_cell.rs

```diff
@@ -218,8 +218,10 @@ impl<R: Region + Clone> WritePass<R> {
 
     /// Consume this regioned write-pass, possibly yielding the parent region back.
     ///
-    /// If all other split descendents of the parent write pass have been dropped or consumed,
+    /// If all other split descendents of the parent write pass have been consumed,
     /// this will return a region equivalent to the parent's.
+    ///
+    /// All write-passes need to be consumed before being dropped to yield the parent region.
```

Comment:
I'm ~OK with this but I think a better solution would be to store some fields in an `Option` and call `take` on it in consume and in `Drop`.